components/search-tab-group/search-tab.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { Helmet } from "react-helmet";
 
 export default (Loader, type) => (props) => {
-  const { keywordSearched } = props;
+  const { keywordSearched, helpType } = props;
   type = type.toLowerCase();
 
   return (
     <div className={`search-${type}-tab`}>
       <Helmet><title>{type.charAt(0).toUpperCase()+type.slice(1)} search</title></Helmet>
-      { keywordSearched && <Loader search={keywordSearched} showCounter={true} />}
+      <Loader search={keywordSearched} showCounter={true} help_type={helpType}/>
```

Comment:
Why did the first part of this line go away? I can vaguely imagine why, but it's not entirely clear.

Reply:
I think it's because the  loader would only produce results if a keyword was searched e.g. if you selected something form the dropdown, no results would appear until you used the search bar. This change was made earlier when the dropdown and search worked together, but I think even with them working separately, this still may be necessary 🤔

Reply:
Yup, this makes more sense now